### DISCUSSION TITLE
How to bundle a php file with the rest of my project?

### DISCUSSION THREAD

**Discussion by ClemGG** [webpack/webpack#18931] [Q&A]
Hello, 

I'm still experimenting with the webpack.config.js file, and my project requires to edit files on the server. I'm using a simple php file as backend to open and overwrite the file. However, it is not bundled by default with webpack, and trying to access it in localhost gives me a 404 error. I would like to now if there is an out of the box way to bundle them together or if there is a specific package to install to achieve this.

Thank you for your time.

**Comment by alexander-akait** [comment]
You can use `writeToDisk` - https://webpack.js.org/configuration/dev-server/#devserverdevmiddleware to write chunks in disk, by default we use memory to store them and use JS file as an entrypoint
